jbmc/unit/java_bytecode/java_bytecode_language/language.cpp

```diff
@@ -45,3 +48,48 @@ SCENARIO(
     }
   }
 }
+
+static void use_external_driver(java_bytecode_languaget &language)
+{
+  optionst options;
+  options.set_option("symex-driven-lazy-loading", true);
+  language.set_language_options(options);
+}
+
+TEST_CASE(
```

Comment:
Why not use SCENARIO instead of TEST_CASE (I believe they are synonyms) and add some GIVEN, WHEN and THEN lines to explain what's going on. That would match the other tests in this folder.

Reply:
Done.